### ISSUE TITLE
Slide Notes timer doesn't work

### ISSUE THREAD

**Issue by caijwei** [hakimel/reveal.js#2243]
When I open the Slide Notes browser page (pressing s) the timer doesn't start. I'm using Chrome and I'm getting this error in the console

```javascript
Uncaught ReferenceError: Reveal is not defined
    at getTimings (notes.html:517)
    at setupTimer (notes.html:574)
    at handleConnectMessage (notes.html:413)
    at notes.html:380
```
My speech and plugin are not in the same domain

You can [access the address](https://www.caijw.com/reveal/1613107875995952125.html) to reproduce the problem

**Comment by sandy8111112004** [comment]
Hi, I would like to try this issue #2243

**Comment by hakimel** [comment]
This issue was previously reported in #1922, and fixed by #2104. The fix is currently only available in the dev branch but will make its way out with the next reveal.js release.
